go/libraries/doltcore/sqle/database.go

```diff
@@ -425,7 +425,7 @@ func (db Database) getTableInsensitive(ctx *sql.Context, head *doltdb.Commit, ds
 			}
 		}
 
-		dt, found = dtables.NewUnscopedDiffTable(ctx, db.ddb, head), true
+		dt, found = dtables.NewUnscopedDiffTable(ctx, db.ddb, db.name, head), true
```

Comment:
(minor) it would be nice to pass `db.name` before `db.ddb`, just to match the other `New...` function calls in this method (i.e. lines 430 and 450).